### ISSUE TITLE
How to Extract Training Loss versus Epoch

### ISSUE THREAD

**Issue by tinnguyen96** [PKU-Cloud-Lab/xLearn#372]
Hi! I would like to programmatically extract the train mse_loss incurred after each training epoch. Currently, I am piping the xlearn printing output to a file, but I don't know how to extract the mse_loss from this .txt log. Is there a more straightforward way to save train mse_loss using either the sklearn API or the native Python API? Thank you!
